### PR TITLE
Python version conf.py

### PR DESCRIPTION
**Pull request by joybhallaa**
## PR Summary
@tacaswell  enquired on gitter about adding a check on the python version in **doc/conf.py**, so I have added a check and an error message to **conf.py**, it checks if the python version is 2.7.16 or lower, and then tells the user about it.
## PR Checklist

- [ ] Has Pytest style unit tests
- [x] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [x] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -17,6 +17,9 @@
 import matplotlib
 import sphinx
 
+if sys.version_info <= (2, 7, 16):
```
Comment (dstansby) [inline doc/conf.py]: Would it be best to do `< (3, 0, 0)` to allow for any possible future python 2 releases?

Comment (joybhallaa) [reply doc/conf.py]: Yes you're right, let me update it.

File: doc/conf.py [hunk 2]
```diff
@@ -17,6 +17,9 @@
 import matplotlib
 import sphinx
 
+if sys.version_info <= (3, 0, 0):
+    print("You're using python 2.x, conf.py works with python3+ only.")
```
Comment (QuLogic) [inline doc/conf.py]: This should fail or otherwise exit here, or any Python 3 code later that fails to parse will cause a confusing second error.

**Comment by tacaswell** [comment]
Thank you @joybhallaa !  I think this is your first Matplotlib contribution :tada: hopefully we will hear from you again!

**Pull request merged**